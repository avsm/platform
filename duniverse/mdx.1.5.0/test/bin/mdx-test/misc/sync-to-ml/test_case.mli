val f: unit -> unit

[@@@part "1"] ;;

module A =
struct

  [@@@part "2"] ;;

  type t = Some of int | Many

  [@@@part "3"] ;;

end
