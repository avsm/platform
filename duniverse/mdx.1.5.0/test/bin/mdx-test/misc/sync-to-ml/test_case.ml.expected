x;;

[@@@part "1"] ;;

let filename_param =
  let open Command.Param in
  anon ("filename" %: string)

[@@@part "toto"] ;;

let x = 3;;
let y = 4;;
print_int x;;
print_int y;;

[@@@part "zzz"] ;;

let f = "hello world!"

[@@@part "42"] ;;

let () =
  f x print_int

[@@@part "new-part-not-in-original"] ;;

let g = [ f ]
